### PR TITLE
Improve terminal output

### PR DESCRIPTION
**Pull request by Turbo87**
This PR adds action highlighting and fixes the slightly broken commands rewriting.

Before:

before
 
After:

after

### CODE REVIEW

**Comment by sjurba** [comment]
Thanx for the pull req. Great idea to add colours. But I wonder if we might change the colour scheme.. It works fine with white on black, but if you have black on white terminal (like me) it doesn't work quite as good. Instead of making the keyword stand out more, it stands out less.

image

**Comment by Turbo87** [comment]
@sjurba good point. could you experiment a bit to find out which colors look good in your setup?

**Comment by Turbo87** [comment]
I've adjusted the styling to this now:

screenshot

screenshot

**Comment by sjurba** [comment]
Thats better, a bit kitch, but you can read it at least. :) 
But could you also add support for a -c/--nocolors option to switch them off..?

**Comment by Turbo87** [comment]
@sjurba done

File: lib/editor.js [hunk 1]
```diff
@@ -8,6 +8,8 @@ var lineNum = 0;
 var lastLine;
 var showStatus = global.appconfig.status;
 
+var noColor = (process.argv.indexOf('--no-color') !== -1);
```
Comment (sjurba) [inline lib/editor.js]: I use minimist parse cmd args in index.js and put them in the global.appconfig object. For example like the show status flag..

Comment (Turbo87) [inline lib/editor.js]: Oh sorry, missed that... I'll amend the commit and use that instead.

Comment (Turbo87) [inline lib/editor.js]: done

**Comment by sjurba** [comment]
Super. Just add --no-colors to the cmd args section in the README and we're good. :)

**Comment by Turbo87** [comment]
@sjurba done

**Comment by sjurba** [comment]
Thanx for this! Did you have any other changes in mind before I publish a new release on npm?

**Comment by Turbo87** [comment]
:tada: 

@sjurba I'm pretty happy with the current state now. just noticed that I missed adding Ctrl+Up/Down to the README though.

**Comment by sjurba** [comment]
Cool. I fixed the README directly on master. Will make release soon.

**Pull request merged**